packages/slice-machine/components/FormFields/Input.tsx

```diff
@@ -71,6 +82,12 @@ export const FormFieldInput = ({
   initialValues,
   isDisabled,
 }: FormFieldInputProps) => {
+  const style = meta.error
+    ? InputFieldStyles.ERROR
+    : isDisabled
```

Comment:
is it possible for there to be a meta.error AND isDisabled?

Reply:
might be, but guess errors should take precedence so that's what we should use in case they both are present